internal/view/assets/less/custom-dialog.less

```
@@ -153,4 +153,23 @@
 			}
 		}
 	}
-}
\ No newline at end of file
+}
+
+
+@media only screen and (max-width: 768px) {
+	.custom-dialog-overlay {
+		padding: 0px;
+	}
+
+	.custom-dialog {
+		width: 100%;
+		height: 100%;
```

You are right that we can do this as well:

```suggestion
		height: 100%;
		resize: none;
```